### DISCUSSION TITLE
Ability to add response headers from a Processor

### DISCUSSION THREAD

**Discussion by bpolaszek** [api-platform/core#6608] [Q&A] [answered]
Is there an undocumented way, within a `Processor`, to prematurely add response headers or programmatically specify the status code?

**Comment by soyuka** [comment, marked as answer]
You can `return new Response`? It's also possible to add `Get(headers: ['foo' => 'bar'], status: 202)`

**Reply by bpolaszek** [reply]
Hi @soyuka, thanks for the reply.

I didn't know a processor could return a `Response` (and if it's actually a good thing), but why not, I'll give it a try :-) 
Thx!

**Reply by soyuka** [reply]
If you return a processor you bypass everything, maybe that a decoration of RespondProcessor would be even more appropriate.
